src/main/java/com/conveyal/taui/models/OpportunityDataset.java

```diff
@@ -0,0 +1,34 @@
+package com.conveyal.taui.models;
+
+public class OpportunityDataset extends Model {
+    /** The human-readable name of the data source from which this came */
+    public String sourceName;
+    /** The unique id for this source set */
+    public String sourceId;
+
+    /** Bucket name */
+    public String bucketName;
+
+    /** Bounds */
+    public int north;
+    public int west;
+    public int width;
+    public int height;
+
+    /** Total Opportunities */
+    public double totalOpportunities;
```

Comment:
This seems handy.  Do we use it somewhere yet?

Reply:
Not yet. But I had a dataset that was giving me crazy numbers and I wasn't sure what the total was. Figured it was worth calculating and storing for easy glancing of correctness.